support-frontend/assets/components/consentBanner/consentBanner.jsx

```diff
@@ -1,79 +1,83 @@
 // @flow
-import React from 'react';
-import type { Dispatch } from 'redux';
-import { connect } from 'react-redux';
-import type { ThirdPartyTrackingConsent } from '../../helpers/tracking/thirdPartyTrackingConsent';
+import React, { Component } from 'react';
+import { getTrackingConsent, writeTrackingConsentCookie, type ThirdPartyTrackingConsent } from '../../helpers/tracking/thirdPartyTrackingConsent';
```

Comment:
could just be `from 'helpers/tracking...` I think, I realise this was already here though so up to you if you want to tidy it up